Ownership/ownership.cc

```diff
@@ -1,16 +1,30 @@
 #include <vector>
 #include <utility>
 
+typedef Image std::vector<char>;
 
 // loadImageFromDisk
-
+Image loadImageFromDisk() {
+	return std::vector<char>(640 * 480);
+}
 
 // saveImageToDisk
-
+void saveImageToDisk(const Image& imgData) {
+	
+}
 
 // setAllPixelsToBlack
-
+/*
+void setAllPixelsToBlack(Image& imgData) {
+	
+}
+*/
+Image setAllPixelsToBlack(Image imgData) {
+	
+}
 
 int main() {
-
+	auto img = loadImageFromDisk();
+	auto img2 = setAllPixelsToBlack(std::move(img));
+	saveImageToDisk(img);
```

Comment:
This will crash since `img` is a moved-from object. Did you mean to use `img2` here?

(This is exactly one of those situations where Rust would have warned you)